cunumeric/runtime.py

```diff
@@ -179,20 +179,15 @@ def _register_dtypes(self) -> None:
                 np.dtype(numpy_type), core_type  # type: ignore
             )
 
-        for n in range(1, LEGATE_MAX_DIM + 1):
-            self._register_point_type(n)
+        for dtype in _CUNUMERIC_DTYPES:
+            type_system.add_type(dtype.type, dtype.size, dtype.code)
 
-    def _register_point_type(self, n: int) -> None:
+    def get_point_type(self, n: int) -> np.dtype[Any]:
         type_system = self.legate_context.type_system
-        point_type = "Point" + str(n)
-        if point_type not in type_system:
-            code = CuNumericTypeCodes.CUNUMERIC_TYPE_POINT1 + n - 1
-            size_in_bytes = 8 * n
-            type_system.add_type(point_type, size_in_bytes, code)
-
-    def get_point_type(self, n: int) -> str:
-        type_system = self.legate_context.type_system
-        point_type = "Point" + str(n)
+        if n == 1:
+            point_type = np.dtype("i8")
+        else:
+            point_type = _CUNUMERIC_DTYPES[n - 2].type
```

Comment:
I would prefer something like this:

```
point_type = np.dtype(",".join(("i8",) * n))
```

The current code depends on the order of entries in `_CUNUMERIC_DTYPES`, and it's possible that someone will change that enum down the line, but neglect to update this code.